backend/src/main/java/moaon/backend/article/service/ArticleService.java

```diff
@@ -61,4 +70,35 @@ public void increaseClicksCount(long id) {
                 .orElseThrow(() -> new CustomException(ErrorCode.ARTICLE_NOT_FOUND));
         article.addClickCount();
     }
+
+    @Transactional
+    public void save(List<ArticleCreateRequest> requests) {
+        for (ArticleCreateRequest request : requests) {
+            Project project = projectRepository.findById(request.projectId()).orElseThrow(
+                    () -> new CustomException(ErrorCode.PROJECT_NOT_FOUND)
+            );
+
+            Article article = new Article(
+                    request.title(),
+                    request.summary(),
+                    articleContentRepository.findByUrl(request.url())
+                            .map(ArticleContent::getContent)
+                            .orElse(""),
+                    request.url().toString(),
+                    LocalDateTime.now(),
+                    project,
+                    Sector.of(request.sector()),
+                    request.topics()
+                            .stream().
+                            map(Topic::of).
+                            toList(),
```

Comment:
클린코드 관련 컨벤션 웬만한건 다 넘기는데 이건 못참겠네요..ㅋㅋ

```suggestion
                    request.topics()
                            .stream()
                            .map(Topic::of)
                            .toList(),
```